frontend/src/pages/Risk/Risk.tsx

```diff
@@ -458,6 +528,28 @@ const Risk: React.FC = (props) => {
     (a, b) => new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime()
   );
 
+  // create severity object for Open Vulnerability chips
+  const severities: VulnSeverities[] = [
+    { label: 'All', sevList: ['', 'Low', 'Medium', 'High', 'Critical'] },
+    { label: 'Critical', sevList: ['Critical'] },
+    { label: 'High', sevList: ['High'] },
+    { label: 'Medium', sevList: ['Medium'] },
+    { label: 'Low', sevList: ['Low'] }
+  ];
+
+  console.log(stats);
```

Comment:
Can you remove this line? It's slowing things down since it's logging everything